.github/workflows/tests.yml

```diff
@@ -182,22 +182,24 @@ jobs:
           fetch-depth: 0
 
       - name: Setup Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v1
+        uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
 
+      # is vc for python 2.7 still relevant?
       - name: Install vcpython27.msi
         run: |
           $wc = New-Object net.webclient
           $wc.Downloadfile("https://github.com/GeoNode/geonode-win-installer/raw/ffb76c7cbf1d6b4970c6c25f79c3c7682a3aa035/VCForPython27.msi", "VCForPython27.msi")
           Start-Process "VCForPython27.msi" /qn -Wait
         shell: pwsh
 
+      # would be great to cache this
       - name: Install miniconda
         run: |
           rmdir C:\Strawberry /s /q
           choco install wget visualstudio2017-workload-vctools
-          wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe
+          wget -q https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe
```

Comment:
Quiet. This download is pretty slow. Why not use New-Object net.webclient as above?